### ISSUE TITLE
Be able to change a AAD app ("AzureAD" in appsettings.json and Startup.cs) to B2C and reciprocally

### ISSUE THREAD

**Issue by jmprieur** [jmprieur/app-provisonning-tool#17]
## Why?
Visual Studio 16.9 will create apps which are SingleOrg not calling anything, and then the app can be configured to add MIcrosoftGraph, calling a downstream API, 

## What?
If the developer chooses a --tenant-id which is a B2C tenant, the `appsettings.json` and `startup.cs` should change to replace "AzureAD" by "AzureAdB2C", or add a new section.

## To be decided
Do we want to ask the question if this is what the user intended?, and an option --force, or --yes

## Repro

1. New app

```Shell
dotnet new webapp --auth SingleOrg
ms-identity-app --tenant-id fabrikamb2c.onmicrosoft.com
> the code is currently configured as an AAD application whereas the tenant is a B2C tenant. 
> Use --yes to confirm that your intent is to change the app to a B2C app

ms-identity-app --tenant-id fabrikamb2c.onmicrosoft.com --yes
```

2. Configure from an existing app.
- if the `tenantId` is a B2C, and the code calls graph, a warning should be emitted tell the user to remove Graph.
- if the `tenantId` is a B2C, and the app is a web API and code calls a downstream API, a warning should be emitted tell the user to remove that call.

**Comment by jennyf19** [comment]
current experience: 
- [ ] create a singleOrg app, specify --tenant-id fabrikamb2c.onmicrosoft.com
![image](https://user-images.githubusercontent.com/19942418/108642697-ac825280-745b-11eb-8317-2660542984b5.png)

**Comment by jennyf19** [comment]
Included in [1.7.0 release](https://github.com/AzureAD/microsoft-identity-web/releases/tag/1.7.0)
